rmw_opendds_shared_cpp/include/rmw_opendds_shared_cpp/node.hpp

```diff
@@ -28,6 +28,12 @@ create_node(
   size_t domain_id,
   const rmw_node_security_options_t * options);
 
+
+RMW_OPENDDS_SHARED_CPP_PUBLIC
+        rmw_ret_t
```

Comment:
```suggestion
rmw_ret_t
```

Reply:
Style updates are helpful. We can commit but we will run a beautify and linter at some point later in the development.

Reply:
This isn't a style thing, this is a typo.

Reply:
This is just a typo.

Reply:
Missed that. Thanks.